src/main/java/se/sundsvall/supportmanagement/service/RevisionService.java

```diff
@@ -198,7 +198,8 @@ public DifferenceResponse compareErrandRevisionVersions(String errandId, int sou
 	public List<Revision> getNoteRevisions(String errandId, String noteId) {
 		verifyExistingErrand(errandId);
 
-		return ErrandNoteMapper.toRevisions(notesClient.findAllNoteRevisions(noteId));
+		//TODO Change to municipalityId when added to API
```

Comment:
Not sure I understand the TODO comments in this class.

Reply:
Municipality is missing in Revision-resources. The scope for this task is to only change integration I think and therefore this temporarly hard-coding.